### ISSUE TITLE
customize time to idle

### ISSUE THREAD

**Issue by bgrayburn** [mizzao/meteor-user-status#74]
sorry if I'm missing a way to do this already, but it'd be great to be able to set the amount of time inactive before switching status to idle.

**Comment by mizzao** [comment]
Yes, there is - `UserStatus.startMonitor(options)` and specify `options.threshold`.

**Comment by bgrayburn** [comment]
thanks! sorry, swear I read the f*@ manual but am too tired.

**Comment by mizzao** [comment]
No problem, it was a very curt note.

**Comment by bgrayburn** [comment]
@mizzao, had you reopened this intentionally? otherwise I'm good on this, thanks again!

**Comment by mizzao** [comment]
Reopened it so that others wouldn't ask the same question.
